README.md

```diff
@@ -47,6 +47,7 @@ You will need a plugin manager to install the plugins and prompts listed on this
 
 - [Official Documentation](http://fishshell.com/docs/current/index.html)
 - [fish cookbook](https://github.com/brj/fish-cookbook) - Tips and recipes for fish-shell users
+- [fish Tutorial](https://github.com/ellerbrock/tutorial-fish-shell-setup-osx) - How to Setup fish Shell with fisherman, Powerline Fonts, iTerm2 and Budspencer Theme on OS X.  
```

Comment:
yes its confusing because the contribution guidelines changes from project to project (often these "titlelized" style). if its ok to you guys you could commit the change and just edit the file how you like. i guess you also have to whitelist those domains:
https://github.com/brj/fish-cookbook
https://www.reddit.com/r/fishshell/
in your travis file => https://travis-ci.org/brj/awesome-fish/builds/133684753 by the way, brj are you the creator of the fish shell?